dashboard/db/migrate/20150507195442_multiple_cohorts_to_workshop.rb

```diff
@@ -0,0 +1,8 @@
+class MultipleCohortsToWorkshop < ActiveRecord::Migration
+  def change
+    create_table :workshops_cohorts do |t|
```

Comment:
I think this would be better named WorkshopCohorts since it's a real model object not an invisible join table (DistrictsCohorts is like that because it started out as a join table)